db/models/test/review.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const db = require('APP/db')
+const Review = require('APP/db/models/Review')
+const {expect} = require('chai')
+
+describe('Review', () => {
+  before('wait for the db', () => db.didSync)
+  const exampleReview = {
+    authorName: 'Susan McGee',
+    title: 'Sun = fun',
+    description: "The sun that I ordered was quite nice, round, and pleasant. and no i'm not talking about my husband",
+    rating: '5',
+    authorId: 3,
```

Comment:
Having the foreign key fields here may break your tests once you have removed the column declarations from the review.js model file, but once we add in the associations the tests should pass... I think.